### PR TITLE
ld: add Indonesian translation

### PR DESCRIPTION
**Pull request by apraditya**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.id/common/ld.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# ld
+
+> Menghubungkan file-file obyek berbarengan.
+> Informasi lebih lanjut: <https://sourceware.org/binutils/docs-2.38/ld.html>.
+
+- Menghubungkan file obyek tertentu yang tidak memiliki kebergantungan ke suatu perintah:
```
Comment (kbdharun) [inline pages.id/common/ld.md:6]: Shouldn't the last work be executable instead of command?

File: pages.id/common/ld.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`ld {{jalan/ke/file.o}} --output {{jalan/ke/output_perintah}}`
```
Comment (kbdharun) [inline pages.id/common/ld.md:8]:
```suggestion
`ld {{jalan/menuju/file.o}} --output {{jalan/menuju/output_perintah}}`
```

File: pages.id/common/ld.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- Menghubungkan 2 file obyek berbarengan:
+
+`ld {{jalan/ke/file1.o}} {{jalan/ke/file2.o}} --output {{jalan/ke/output_executable}}`
```
Comment (kbdharun) [inline pages.id/common/ld.md:12]:
```suggestion
`ld {{jalan/menuju/file1.o}} {{jalan/menuju/file2.o}} --output {{jalan/menuju/output_executable}}`
```

File: pages.id/common/ld.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+- Menghubungkan secara dinamis sebuah program x86_64 ke glibc (jalan-jalan file berubah bergantung pada sistem):
+
+`ld --output {{jalan/ke/output_executable}} --dynamic-linker /lib/ld-linux-x86-64.so.2 /lib/crt1.o /lib/crti.o -lc {{jalan/ke/file.o}} /lib/crtn.o`
```
Comment (kbdharun) [inline pages.id/common/ld.md:16]:
```suggestion
`ld --output {{jalan/menuju/output_executable}} --dynamic-linker /lib/ld-linux-x86-64.so.2 /lib/crt1.o /lib/crti.o -lc {{jalan/menuju/file.o}} /lib/crtn.o`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages.id/common/ld.md (hunk 1 above)
Comment (apraditya) [reply pages.id/common/ld.md:6]: Agree. What's the proper term for executable (should it be translated)?

Comment (kbdharun) [reply pages.id/common/ld.md]: I am not a native speaker. Also, I too couldn't find a word for it online. Maybe similar to file we can leave executable as it is. What do you think?

**Review by kbdharun — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**